### PR TITLE
Demo and document new inline "clear" button

### PR DESCRIPTION
**Pull request by ntdb**
Dependent upon #165.

screen shot 2015-08-14 at 1 43 04 pm

### CODE REVIEW

**Comment by martijnrusschen** [comment]
Hi @ntdb! Great stuff! See my inline comment to make sure this works fine!

File: style.css [hunk 1]
```diff
@@ -163,6 +167,42 @@
   border-color: #aeaeae;
   box-shadow: inset 0 2px 2px #e9e9e9, 0 0 10px 0 rgba(73, 107, 125, 0.3);
 }
+.datepicker__input:not(:valid) ~ .close-icon {
```
Comment (martijnrusschen) [inline style.css]: Looks like you've writing this directly into the compiled css, there should be a .scss file that compiles into this css file.

Comment (ntdb) [inline style.css]: @martijnrusschen it's actually in `datepicker.scss` on [my other PR](https://github.com/Hacker0x01/react-datepicker/pull/165/files). It's also here thanks to webpack. :)

Comment (martijnrusschen) [inline style.css]: nvm, looks didn't check the other PR out yet. Should be fine now.

**Pull request merged**